### ISSUE TITLE
Console gets spammed with exceptions

### ISSUE THREAD

**Issue by MartinSStewart** [layflags/elm-monitor#5]
After setting up elm-monitor I get the following exception in the console repeatedly
```Expecting Keyword True at row 1, col 1
Expecting Keyword False at row 1, col 1
Expecting Symbol () at row 1, col 1
Expecting Symbol ' at row 1, col 1
Expecting Symbol " at row 1, col 1
Expecting Symbol < at row 1, col 1
Expecting { at row 1, col 1
Expecting [ at row 1, col 1
Expecting Keyword True at row 1, col 116
Expecting Keyword False at row 1, col 116
Expecting Symbol () at row 1, col 116
Expecting Symbol ' at row 1, col 116
Expecting Symbol " at row 1, col 116
Expecting Symbol < at row 1, col 116
Expecting { at row 1, col 116
Expecting [ at row 1, col 116
Expecting Symbol ( at row 1, col 116
Expecting Variable at row 1, col 116
Expecting Number at row 1, col 116
Expecting Variable at row 1, col 1
Expecting Number at row 1, col 1
```
I've used elm-monitor with another Elm project and I haven't seen the exception there. I'm not sure what the difference is that causes it to appear here.  This exception appears both on Firefox and Chrome.

Edit: The frequency of these exceptions seems to match when a certain Elm msg is received. In case it helps, this is what elm-monitor writes out for that msg
```
(LongPollRequest (Err NetworkError),{ currentTime = Posix 1551788676113, hideErrorOverlay = False, lastMessageId = -1, messages = [], quietForOneSecond = Debouncer (Config { accumulator = <function>, emitWhenUnsettled = Nothing, emitWhileUnsettled = Nothing, settleWhenQuietFor = Just 1000 }) Settled, releaseBuilds = Loading, serverData = Failure NetworkError, uploadTimeRequest = Nothing })
```

**Comment by layflags** [comment]
Thx for reporting this issue @MartinSStewart. @ChristophP do you have an idea why the parser fails here?

**Comment by ChristophP** [comment]
@MartinSStewart Thanks for reporting I'll take a look

**Comment by layflags** [comment]
@MartinSStewart The bug should be fixed now thx to @ChristophP 👍 I published v0.0.8

**Comment by MartinSStewart** [comment]
I tested running 0.0.8 and I can confirm it works. Thanks!
